packages/@lightningjs/ui-components/src/components/Artwork/Artwork.styles.js

```diff
@@ -29,7 +29,7 @@ export const base = theme => ({
   fallbackSrc: undefined,
   fillColor: theme.color.overlay,
   gradientColor: theme.color.material,
-  imageScale: undefined,
```

Comment:
Tile has a failing test that needs to be updated because of this change.

Reply:
Test is passing now. Updated the test to check for 1 since that is the Artwork's default